orc8r/cloud/go/test_utils/service.go

```diff
@@ -89,6 +112,24 @@ func NewTestOrchestratorService(
 	return srv, lis
 }
 
+// NewTestOrchestratorServiceWithControlProxy create a Orchestrator Service and a
```

Comment:
NewTestOrchestratorServiceWithControlProxy createS ... This service will configure a custom ... Remember to delete the temporary file once the test is finished by calling os.RemoveAll(dir)